bears/js/JSHintBear.py

```diff
@@ -308,6 +306,12 @@ def generate_config(filename, file,
             This option is used to specify the ECMAScript version to which the
             code must adhere to.
         """
+        # Assume that when es_version is bool, it is intended for the
+        # deprecated use_es6_version
+        if es_version is True:
+            es_version = 6
+        elif es_version is False:
+            es_version = 5
```

Comment:
actually you can put this into the conversion routine/annotation for es_version huh?